website/projects/scimaps/src/app/shared/news-item/news-item.model.ts

```diff
@@ -0,0 +1,20 @@
+export interface NewsItem {
+
+  /** Title of news item */
+  title: string;
+
+  /** Date of publication */
+  date: string;
```

Comment:
This should be of type `Date` and then use the date pipe in the template to format correctly